src/Features/Core/Portable/xlf/FeaturesResources.de.xlf

```diff
@@ -119,97 +112,87 @@
       </trans-unit>
       <trans-unit id="Indent_all_arguments">
         <source>Indent all arguments</source>
-        <target state="new">Indent all arguments</target>
         <note />
       </trans-unit>
       <trans-unit id="Indent_all_parameters">
         <source>Indent all parameters</source>
-        <target state="new">Indent all parameters</target>
+        <target state="translated">Alle Parameter einrücken</target>
         <note />
       </trans-unit>
       <trans-unit id="Indent_wrapped_arguments">
         <source>Indent wrapped arguments</source>
-        <target state="new">Indent wrapped arguments</target>
         <note />
       </trans-unit>
       <trans-unit id="Indent_wrapped_parameters">
         <source>Indent wrapped parameters</source>
-        <target state="new">Indent wrapped parameters</target>
+        <target state="translated">Einzug für umschlossene Parameter</target>
```

Comment:
This sounds more like an action than the statement it is now, and "einrücken" would be a better verb here.